package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "scripts": {
     "build": "babel src -d lib",
-    "dev": "babel -w src -d lib",
+    "dev": "npm run build -- --watch",
```

Comment:
This change is unrelated to the main change in this PR. I decided it was too small to merit being pushed as a separate PR, let me know if you disagree.